scripts/generate_typedoc.sh

```diff
@@ -10,25 +10,31 @@
 #
 ###
 
-zoweVersion=$1
-imperativeVersion=$2
-cliVersion=$3
```

Comment:
why didn't we add the `v` before? 😋

Reply:
Not really any good reason 😛 I think it was because `v` was hardcoded in places where the variables were used below, and now it's not in the case of "next".